### DISCUSSION TITLE
How to use    rpl-border-router mote with using TSCH ?  Please ,need your help!

### DISCUSSION THREAD

**Discussion by Mbaron12** [contiki-ng/contiki-ng#2388] [Q&A] [answered]
Hi , sir !
IF the rpl-border-router mote use tsch ,the rpl-border-router mote can work right ? My rpl-border-router mote which use tsch can not work right. It didn't own ipv6 adress. How to use TSCH  on the   rpl-border-router mote?  Or how to use  **coap-example-server** and   **rpl-border-router** mote with using TSCH ?
How can I solve this problem? Looking forward to your reply!
I added the following to  **Makefile**:
MAKE_MAC = MAKE_MAC_TSCH

**contiki-ng-develop/examples/rpl-border-router-1$** sudo make TARGET=cc2538dk connect-router
sudo ../../tools/serial-io/tunslip6 fd00::1/64
********SLIP started on /dev/ttyUSB0'' opened tun device /dev/tun0''
ifconfig tun0 inet hostname mtu 1500 up
ifconfig tun0 add fd00::1/64
ifconfig tun0 add fe80::0:0:0:1/64
ifconfig tun0

tun0: flags=4305<UP,POINTOPOINT,RUNNING,NOARP,MULTICAST> mtu 1500
inet 127.0.1.1 netmask 255.255.255.255 destination 127.0.1.1
inet6 fe80::b4b9:8473:81c5:2802 prefixlen 64 scopeid 0x20
inet6 fd00::1 prefixlen 64 scopeid 0x0
inet6 fe80::1 prefixlen 64 scopeid 0x20
unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00 txqueuelen 500 (UNSPEC)
RX packets 0 bytes 0 (0.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 0 bytes 0 (0.0 B)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

**Comment by Mbaron12** [comment, marked as answer]
I solve it!

**Reply by Blendinl** [reply]
@Mbaron12 Do you mind sharing how you fixed it?

**Reply by hmnatividad** [reply]
Can you share how you solved it? i encountered the same problem.
